src/components/List/ListItem/Variants/SuggestionItem/SuggestionItem.scss

```diff
@@ -0,0 +1,73 @@
+.SuggestionItem {
+    all: unset;
+    display: flex;
+    text-decoration: none;
+    padding: 10px var(--space-150);
```

Comment:
we shouldn't use 10px. I will check with Dima, what it should be.

Reply:
done